static/src/javascripts/projects/common/modules/identity/forms.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import bean from 'bean';
+import bonzo from 'bonzo';
+
+export const forgottenEmail = (): void => {
+    let hashEmail;
+    let input;
+
+    if (document.body) {
+        const form = document.body.querySelector('.js-reset-form');
+        if (form) {
+            hashEmail = window.location.hash.match('email=([^&#]*)');
+            input = form.querySelector('.js-reset-email');
+            if (hashEmail && input && input instanceof HTMLInputElement) {
+                input.value = hashEmail[1];
+            }
+        }
+    }
+};
+
+export const passwordToggle = (): void => {
+    let password;
+    let toggleClass;
+    let toggleTmpl;
+    let $toggle;
+
+    if (document.body) {
+        const form = document.body.querySelector('.js-register-form');
+        if (form) {
+            password = form.querySelector('.js-register-password');
+            toggleClass = 'js-toggle-password';
+            toggleTmpl =
+                `${'<div class="form-field__note form-field__note--right mobile-only">' +
+                    '<a href="#toggle-password" class="'}${toggleClass}" data-password-label="Show password"` +
+                ` data-text-label="Hide password" data-link-name="Toggle password field">Show password</a>` +
+                `</div>`;
+            $toggle = bonzo(bonzo.create(toggleTmpl)).insertBefore(password);
```

Comment:
You could move all of these assignments to the top and make 

```
let password;
let toggleClass;
let toggleTmpl;
let $toggle;
```

`const`s.

Reply:
yep, will do. ta